source/json/people.schema.json

```diff
@@ -9,8 +9,7 @@
         "chance": "name"
       },
       "role": {
-        "type": "string",
-        "pattern": "Staff|OWF Host|Partner|Hive NYC Partner|Advisor|Member, MLN Advisory Group"
```

Comment:
I can fix that if we have a specified list of acceptable roles and we want to confine it to those
cc @hannahkane